Update tesla-style-solar-power-card.js

pman07 opened the pull request:
Fixed Issue #51. Improve SVG calculations for power line and SVG window alignment/sizing/re-sizing.

reptilex commented:
Hi. Thanks a lot. That corrections seems pretty legit. But the problem is your are changing the compiled js file. For me to add your correction to the actual code you need to change the typescript functions not the javascript. This is just transpiled typescript. Install the project via NPM and change the ts files inside the src folder. Than transpile/compile and you will have made the actual changes.
Probably this is the file you are interested in:
https://github.com/reptilex/tesla-style-solar-power-card/blob/master/src/services/HtmlResizeForPowerCard.ts

pman07 commented:
I played around with that a bit, I think that's a bit beyond me currently.
At least getting it to compile on windows... I found all the files and
updated them, I can still upload the files I changed, but I can't prove it
will compile.

I'll keep messing with it, just not going to be very quick lol.

On Mon, Jun 14, 2021 at 1:39 AM reptilex ***@***.***> wrote:

> Hi. Thanks a lot. That corrections seems pretty legit. But the problem is
> your are changing the compiled js file. For me to add your correction to
> the actual code you need to change the typescript functions not the
> javascript. This is just transpiled typescript. Install the project via NPM
> and change the ts files inside the src folder. Than transpile/compile and
> you will have made the actual changes.
> Probably this is the file you are interested in:
>
> https://github.com/reptilex/tesla-style-solar-power-card/blob/master/src/services/HtmlResizeForPowerCard.ts
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/reptilex/tesla-style-solar-power-card/pull/71#issuecomment-860419413>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANMRXBD4KTOXGHUCKSCC57TTSWP25ANCNFSM46T4AT7Q>
> .
>

pman07 commented:
Nevermind, I had a breakthrough on learning how to compile. Should be good
now, submitted a new pull request.

On Mon, Jun 14, 2021 at 2:25 PM Michael Pierson ***@***.***> wrote:

> I played around with that a bit, I think that's a bit beyond me currently.
> At least getting it to compile on windows... I found all the files and
> updated them, I can still upload the files I changed, but I can't prove it
> will compile.
>
> I'll keep messing with it, just not going to be very quick lol.
>
> On Mon, Jun 14, 2021 at 1:39 AM reptilex ***@***.***> wrote:
>
>> Hi. Thanks a lot. That corrections seems pretty legit. But the problem is
>> your are changing the compiled js file. For me to add your correction to
>> the actual code you need to change the typescript functions not the
>> javascript. This is just transpiled typescript. Install the project via NPM
>> and change the ts files inside the src folder. Than transpile/compile and
>> you will have made the actual changes.
>> Probably this is the file you are interested in:
>>
>> https://github.com/reptilex/tesla-style-solar-power-card/blob/master/src/services/HtmlResizeForPowerCard.ts
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/reptilex/tesla-style-solar-power-card/pull/71#issuecomment-860419413>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ANMRXBD4KTOXGHUCKSCC57TTSWP25ANCNFSM46T4AT7Q>
>> .
>>
>
